src/Error/ErrorHandler.php

```diff
@@ -136,7 +136,6 @@ protected function _displayException(Throwable $exception): void
                 Router::getRequest()
             );
             $response = $renderer->render();
-            $this->_clearOutput();
```

Comment:
Curious why this was never called.

Reply:
The ErrorHandlerMiddleware now gets an instance of the ExceptionRenderer and calls render() on it directly and no longer calls handleException().